src/console/Microsoft.Health.Fhir.Ingest.Console.csproj

```diff
@@ -1,36 +1,35 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+	</PropertyGroup>
+	<PropertyGroup>
+		<IsPackable>true</IsPackable>
+	</PropertyGroup>
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="4.0.4" />
+		<PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.0.1">
+			<PrivateAssets>all</PrivateAssets>
+			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+		</PackageReference>
+		<PackageReference Include="Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers" Version="0.3.261602">
+			<PrivateAssets>all</PrivateAssets>
+		</PackageReference>
+	</ItemGroup>
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-  </PropertyGroup>
+	<ItemGroup>
+		<ProjectReference Include="..\lib\Microsoft.Health.Common\Microsoft.Health.Common.csproj" />
+		<ProjectReference Include="..\lib\Microsoft.Health.Events\Microsoft.Health.Events.csproj" />
+		<ProjectReference Include="..\lib\Microsoft.Health.Fhir.Ingest\Microsoft.Health.Fhir.Ingest.csproj" />
+		<ProjectReference Include="..\lib\Microsoft.Health.Fhir.R4.Ingest\Microsoft.Health.Fhir.R4.Ingest.csproj" />
+		<ProjectReference Include="..\lib\Microsoft.Health.Expressions\Microsoft.Health.Expressions.csproj" />
+	</ItemGroup>
 
-  <PropertyGroup>
-    <IsPackable>true</IsPackable>
-  </PropertyGroup>
+	<ItemGroup>
+		<None Update="local.appsettings.json">
+			<CopyToOutputDirectory>Always</CopyToOutputDirectory>
+			<CopyToPublishDirectory>Never</CopyToPublishDirectory>
+		</None>
+	</ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="4.0.4" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="3.1.12" />

```

Comment:
Looks like this reference was dropped.  Is it still needed?

Reply:
No the Microsoft.Extensions.Hosting package was identified as transient during the migration so I removed it. I've also verified both the console app and the App Service running the Azure Functions are running successfully. 

Feel free to check it out: [test-net-upgrade-2](https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/4092fd42-145e-4e91-aa15-96f66d2b6deb/resourceGroups/test-net6-upgrade-2/overview)